lib/protobuf/nats.rb

```diff
@@ -50,6 +50,7 @@ def self.subscription_key(service_klass, service_method)
     def self.start_client_nats_connection
       @start_client_nats_connection ||= begin
         GET_CONNECTED_MUTEX.synchronize do
+          return if @client_nats_connection
```

Comment:
We received an error where we were trying to create a message queue with the java-nats connection = nil. I'm not really sure how this happened but maybe the instance var wasn't set before the next thread came in? If so, then this extra instance var check should work since it's explicitly set in the synchronize block. This is a guess, really.